mlflow/tracking/client.py

```diff
@@ -255,6 +647,37 @@ def delete_tag(self, run_id, key):
 
         :param run_id: String ID of the run
         :param key: Name of the tag
+
+        .. code-block:: python
+            :caption: Example
+
+            from mlflow.tracking import MlflowClient
+
+            def print_run_info(run):
+                print("run_id: {}".format(run.info.run_id))
+                print("Tags: {}".format(run.data.tags))
+
+            # Create a run with some tags under the default experiment id "0"
```

Comment:
This also sounds a bit strange because we create a run under an experiment, not an ID.

Reply:
The left-out proposition "with" is redundant; it's just inferred. Now one can be a bit formal a write:
"Create a run with some tags under default experiment whose default id is '0'."  Or
"Create a run with some tags under an experiment with default experiment id '0'.